### PR TITLE
Feature/menu and tests

### LINKED ISSUE #8: Menu [2pts]
- [x] Create a navigation menu that allows users to open all of the pages you created.

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
In this pull request, I have done the following:

- [x] Created a navigation menu that allows users to open all of the pages you created.
- [x] Added test files

### CODE REVIEW

File: spec/system/foods_spec.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                         quantity: 10)
+    @foods = Food.all
+  end
+
+  before(:each) do
+    sign_in User.create!(name: 'John',
+                         email: 'john.doe@mail.com',
+                         password: 'admin1234',
+                         password_confirmation: 'admin1234',
+                         confirmed_at: Time.now)
```
Comment (andrianarivo) [inline spec/system/foods_spec.rb]: You can improve this spec:
`sign_in @owner`

On the before(:all)
```
@owner = User.first
@owner ||=  User.create!(name: 'John',
                         email: 'john.doe@mail.com',
                         password: 'admin1234',
                         password_confirmation: 'admin1234',
                         confirmed_at: Time.now)
```

**Review by andrianarivo — CHANGES_REQUESTED** [review]
### Hi @rachaouldbabaali  , 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Followed git workflow
- No linter errors

## Required Changes ♻️

-  Please make sure all your tests are passing
image

###  Happy coding!👏👏👏

**Review by andrianarivo — APPROVED** [review]

**Review by andrianarivo — APPROVED** [review]
## Hi @rachaouldbabaali  , 

The project setup is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Happy coding!👏👏👏


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rachaouldbabaali** [comment]
Thank you @andrianarivo, I truly appreciate your time and efforts in reviewing

**Pull request merged**